website/docs/r/codeartifact_domain.html.markdown

```diff
@@ -30,8 +30,9 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The Name of Domain.
-* `arn` - The ARN of Domain.
+* `id` - The ARN of the Domain.
+* `arn` - The ARN of the Domain.
```

Comment:
This looks good based on [the underlying code](https://github.com/hashicorp/terraform-provider-aws/blob/bf0941493da9a86098ab735fbe2150189db829ad/aws/resource_aws_codeartifact_domain.go#L87) 👍